published/external/xdpapi_helper.h

```diff
@@ -0,0 +1,68 @@
+#pragma once
+
+#include "xdpapi.h"
+
+#define TEST_TIMEOUT_ASYNC_MS 1000
+#define POLL_INTERVAL_MS 10
```

Comment:
name needs to be scoped - probably with `_` prefix, or `#undef`'d at the end of this header